src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/template-literal-expression/code.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import React, { type Node } from 'react';
+import { tailwind } from '@adeira/sx-tailwind';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
You could maybe add an Eslint config file and disable this rule for these fixtures. Just a nit though. 🙂